npm/src/compiler/compiler.ts

```diff
@@ -15,22 +15,43 @@ type Wasm = typeof import("../../lib/node/qsc_wasm.cjs");
 // for running the compiler in the same thread the result will be synchronous (a resolved promise).
 export interface ICompiler {
   checkCode(code: string): Promise<VSDiagnostic[]>;
-  getHir(code: string): Promise<string>;
+  getHir(code: string, language_features?: string[]): Promise<string>;
+  /** @deprecated -- switch to using `ProgramConfig`-based overload **/
```

Comment:
Can you make this string _extremely_ verbose and helpful? The first person to read the deprecation error is going to freak out even though we're not actually breaking anything ;)